src/figuresClasses.ts

```diff
@@ -1,19 +1,75 @@
 export interface Figure {
-
+  shape: string;
+  color: string;
+  getArea():number
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  shape = 'triangle';
+
+  constructor(
+    public color: string,
```

Comment:
Here should be enum with colors
```suggestion
    public color: Color,
```

Reply:
I pushed what I mean, so you can look at this

Reply:
agree , your version is better 👍